pronto/api/signature.py

```diff
@@ -647,3 +650,26 @@ def digest(self, obj: dict):
             key = "similarity"
 
         return i, j, -obj["residues"][key], -obj[key]
+
+
+def check_if_ncbifam_amr(method_ac):
+
+    con = utils.connect_oracle()
+    cur = con.cursor()
+    cur.execute(
+        f"""
+        SELECT *
+        FROM INTERPRO.NCBIFAM_AMR
+        WHERE METHOD_AC = '{method_ac}'"""
+    )
+
+    return len(cur.fetchall()) > 0
```

Comment:
- Use [bind variables](https://python-oracledb.readthedocs.io/en/latest/user_guide/bind.html#using-bind-variables) to avoid possible SQL injection
- Return the number of rows (`COUNT(*)`) instead of the full row

```suggestion
        """
        SELECT COUNT(*)
        FROM INTERPRO.NCBIFAM_AMR
        WHERE METHOD_AC = :1
        """,
        [accession]
    )
    cnt, = cur.fetchone()
    return cnt > 0
```